### PR TITLE
Get follower count; Remove followers;

### PR DESCRIPTION
**Pull request by ciprianlujeru**

### CODE REVIEW

File: src/actions/followers.js [hunk 1]
```diff
@@ -173,6 +175,40 @@ export function getFollowers(theUrl, selectedTopics, objKey) {
...
+    // if there is a previous request for the same topic.id
+    if (prevGetFollowersCountRequests[topic.id] !== undefined) {
+      // cancel the previous request
+      prevGetFollowersCountRequests[topic.id].cancel();
+    }
+
+    const queryObj = {topic: topic.id};
+    const options = {method: "GET", "mode": "cors"};
+
+    theUrl = theUrl.replace(":licenceId", config["FOLLOWERS_LICENCE_ID"]);
```
Comment (Thurst-ft) [inline src/actions/followers.js]: Don't really understand why we are doing this. Is this a hack to get round not having the correct licence right now.

File: src/actions/followers.js [hunk 2]
```diff
@@ -173,6 +175,40 @@ export function getFollowers(theUrl, selectedTopics, objKey) {
...
+    prevGetFollowersCountRequests[topic.id] = doRequest(theUrl, options).then((response) => {
+
+      if (response.followers) {
+        Object.keys(response.followers).forEach((type) => {
+          dispatch(updateFollowerCount(topic.id, type, response.followers[type]));
+        });
+      }
+
+    }, (error) => {
+
```
Comment (Thurst-ft) [inline src/actions/followers.js]: How is the error handled here?

**Comment by Thurst-ft** [comment]
Added a couple of comments the first I think well be resolved as part of integrating the licence and admin auth work

File: src/actions/followers.js (hunk 2 above)
Comment (dgem) [inline src/actions/followers.js]: Raised https://jira.ft.com/browse/KMT-125 to address error handling

File: src/actions/followers.js (hunk 1 above)
Comment (dgem) [inline src/actions/followers.js]: Looks like a temp fix until we can access the licence id properly

Comment (ciprianlujeru) [inline src/actions/followers.js]: This is for updating the count for topics whenever the followers are listed. This is what we talked about yesterday. Before this, we only got the follower count when we fetching the topic details, but if we added or removed a follower, the count remained unchanged. For this, I created methods to update the count right after the adding/removal and also when retrieving and displaying the follower list.

File: src/actions/followers.js (hunk 2 above)
Comment (ciprianlujeru) [inline src/actions/followers.js]: The error handling is done in the `doRequest` method. The error condition here is only present for when you might need something to happen when an error occurres (except displaying a notification). For refference please check the `getFollowers` method (from the same file) and also the `doRequest` one to see the error handling.

**Pull request merged**